var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -22,7 +22,9 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+
```

Comment:
it is not there in pretty much any `package.py`. So for consistency i would not add it.